### PR TITLE
Style/single sighting

### PR DESCRIPTION
**Pull request by jasmyn2244**
____ Wrote Tests ___x_ Implemented ____ Reviewed

## Neccesary checkmarks:

    [x] All Tests are Passing in all environments

    [] The code will run locally

## Type of change

    [x] New feature
    [] Bug Fix
    [] Refactor

## Description of Change:

    description: Style Single Sighting Page
    

## Check the correct boxes

    [x] This broke nothing
    [] This broke some stuff
    [] This broke everything

## Testing Changes

    [x] No Tests have been changed
    [] Some Tests have been changed
    [] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:

    [x] My code has no unused/commented out code
    [x] My code has no binding.pry's
    [xx] I have reviewed my code
    [x] I have commented my code, particularly in hard-to-understand areas
    [x] I have fully tested my code

### CODE REVIEW

File: src/Components/SingleSighting.js [hunk 1]
```diff
@@ -29,20 +31,42 @@ const SingleSighting = () => {
     }
   })
 
+
   useEffect(() => {
     setClick(false)
   }, [])
 
+  const hasOverflow = (current) => {
```
Comment (krdurrett) [inline src/Components/SingleSighting.js:39]: Smart idea for conditionally rendering the scroll

File: src/Components/SingleSighting.js [hunk 2]
```diff
@@ -52,18 +76,23 @@ const SingleSighting = () => {
         </div>
       </div>
       <div className='sighting-info-container'>
-      <div className='sighting-details'>
-        <p className='sighting-title'>{data.sightingById.title}</p>
-        <p className='sighting-location'>Location: {data.sightingById.location}</p>
-        <p className='sighting-description'>{data.sightingById.description}</p>
-        <p className='near-text'>Nearby Trails:</p>
+        <div className='sighting-details'>
```
Comment (krdurrett) [inline src/Components/SingleSighting.js:79]: This scroll with the conditional arrow looks great!

**Review by RosaTheDev — APPROVED** [review]

**Review by krdurrett — APPROVED** [review]
Single Sighting looks great!

File: src/Components/SingleCryptid.js [hunk 3]
```diff
@@ -29,6 +29,8 @@ const SingleCryptid = () => {
     }
   })
 
+  console.log(loading, data, error)
```
Comment (GraceGardner) [inline src/Components/SingleCryptid.js:32]: Here's the console log that was missing!

File: src/Components/SingleSighting.js (hunk 2 above)
Comment (GraceGardner) [reply src/Components/SingleSighting.js:79]: ^^^

**Review by GraceGardner — APPROVED** [review]
Styling looks good

**Pull request merged**